tethys_apps/static/tethys_apps/js/app_base.js

```diff
@@ -298,4 +298,26 @@ var TETHYS_APP_BASE = (function() {
 
 }()); // End of package wrapper
 // NOTE: that the call operator (open-closed parenthesis) is used to invoke the library wrapper
-// function immediately after being parsed, returning the public interface object.
\ No newline at end of file
+// function immediately after being parsed, returning the public interface object.
+
+
+// Global Functions for App Util
+
+/*****************************************************************************
+ *
+ * Cross Site Request Forgery Token Configuration
+ *   copied from (https://docs.djangoproject.com/en/1.7/ref/contrib/csrf/)
+ *
+ *****************************************************************************/
+
+function csrfSafeMethod(method) {
```

Comment:
Make this a private method of `TETHYS_APP_BASE`. I don't think this has general use.